packages/ui/src/components/Modal/style.less

```diff
@@ -87,9 +90,54 @@
       padding: @condo-modal-half-padding-without-border @condo-modal-padding @condo-modal-half-padding;
       border-top: @condo-global-border-width-default solid @condo-modal-header-scrolling-border-bottom-color;
 
+      display: flex;
```

Comment:
Are you sure, that it's our responsibility to control footer elements?

Reply:
Since it's not clear:
1. Not obvious that React.Node accept list
2. Not obvious, how should it render it (like first element should be on a right or left side), since all of them are floating to right.

That's why I prefer to use Space there....Then you're 100% sure what's going on